smriprep/workflows/surfaces.py

```diff
@@ -531,9 +529,21 @@ def init_gifti_surface_wf(*, name="gifti_surface_wf"):
         run_without_submitting=True,
     )
     fs2gii = pe.MapNode(
-        fs.MRIsConvert(out_datatype="gii"), iterfield="in_file", name="fs2gii"
+        fs.MRIsConvert(out_datatype="gii",to_scanner=True), iterfield="in_file", name="fs2gii"
+    )
+    surfmorph_list = pe.Node(
+        niu.Merge(3,ravel_inputs=True),
+        name="surfmorph_list",
+        run_without_submitting=True,
+    )
+    fscurv2funcgii = pe.MapNode(
```

Comment:
i'm confused about this step - we aren't doing any functional processing here

Reply:
Whoops. This is for outputting the metric surfaces that will be needed for the "goodvoxels" projection fix to eliminate aliasing in the projected outputs.